eppo_client/configuration_store.py

```diff
@@ -1,33 +1,25 @@
 from typing import Dict, Optional, TypeVar, Generic
-from cachetools import LRUCache
 
 from eppo_client.read_write_lock import ReadWriteLock
 
 T = TypeVar("T")
 
 
 class ConfigurationStore(Generic[T]):
-    def __init__(self, max_size: int):
-        self.__cache: LRUCache = LRUCache(maxsize=max_size)
+    def __init__(self):
+        self.__cache: Dict[str, T] = {}
```

Comment:
How is this cache used? If it's for _assignments_ I think we do want some sort of eviction

Reply:
This cache is used to store the flag configuration. There is no cache for assignments for the Python SDK at the moment.